strawberryfields/gbs/sample.py

```diff
@@ -308,28 +308,15 @@ def vibronic(
 
     **Example usage:**
 
->>> t = np.array([0., 0., 0., 0., 0., 0., 0.])
->>> U1 = np.array(
->>>     [[-0.07985219, 0.66041032, -0.19389188, 0.01340832, 0.70312675, -0.1208423, -0.10352726],
->>>     [0.19216669, -0.12470466, -0.81320519, 0.52045174, -0.1066017, -0.06300751, -0.00376173],
->>>     [0.60838109, 0.0835063, -0.14958816, -0.34291399, 0.06239828, 0.68753918, -0.07955415],
->>>     [0.63690134, -0.03047939, 0.46585565, 0.50545897, 0.21194805, -0.20422433, 0.18516987],
->>>     [0.34556293, 0.22562207, -0.1999159, -0.50280235, -0.25510781, -0.55793978, 0.40065893],
->>>     [-0.03377431, -0.66280536, -0.14740447, -0.25725325, 0.6145946, -0.07128058, 0.29804963],
->>>     [-0.24570365, 0.22402764, 0.003273, 0.19204683, -0.05125235, 0.3881131, 0.83623564]])
->>> r = np.array(
->>>     [0.09721339, 0.07017918, 0.02083469, -0.05974357, -0.07487845, -0.1119975, -0.1866708])
->>> U2 = np.array(
->>>     [[-0.07012006, 0.14489772, 0.17593463, 0.02431155, -0.63151781, 0.61230046, 0.41087368],
->>>     [0.5618538, -0.09931968, 0.04562272, 0.02158822, 0.35700706, 0.6614837, -0.326946],
->>>     [-0.16560687, -0.7608465, -0.25644606, -0.54317241, -0.12822903, 0.12809274, -0.00597384],
->>>     [0.01788782, 0.60430409, -0.19831443, -0.73270964, -0.06393682, 0.03376894, -0.23038293],
->>>     [0.78640978, -0.11133936, 0.03160537, -0.09188782, -0.43483738, -0.4018141, 0.09582698],
->>>     [-0.13664887, -0.11196486, 0.86353995, -0.19608061, -0.12313513, -0.08639263, -0.40251231],
->>>     [-0.12060103, -0.01169781, -0.33937036, 0.34662981, -0.49895371, 0.03257453, -0.70709135]])
->>> alpha = np.array(
->>>     [0.15938187, 0.10387399, 1.10301587, -0.26756921, 0.32194572, -0.24317402, 0.0436992])
->>> vibronic(t, U1, S, U2, alpha, 2, 0.0)
+>>> from strawberryfields.gbs import data, vibronic
+>>> formic = data.Formic()
+>>> w = formic.w
+>>> wp = formic.wp
+>>> Ud = formic.Ud
+>>> delta = formic.delta
+>>> T = 0
+>>> t, U1, r, U2, alpha = vibronic.gbs_params(w, wp, Ud, delta, T)
+>>> vibronic(t, U1, r, U2, alpha, 2, 0.0)
```

Comment:
I realised this codeblock might not work, as it's unclear whether `vibronic` is the module or the function. We implicitly assume in codeblocks that the user has done `from strawberryfields.gbs.currentmodule import *`, but I think this would be problematic here. How about we just do:
```suggestion
>>> sample.vibronic(t, U1, r, U2, alpha, 2, 0.0)
```